recipes/itk-meshtopolydata/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# install using pip from the whl files on PyPI
+
+POST=""
+PYPI_VER="${PY_VER:0:1}${PY_VER:2:3}"
+
+if [ `uname` == Darwin ]; then
```

Comment:
I know it is painful to write these in the recipe itself but we need to have some sort of checksum check and these if-clauses do not ensure we got the right tarball. Take a look at the [recipe for jaxlib](https://github.com/conda-forge/jaxlib-feedstock/blob/master/recipe/meta.yaml) for an example on how to use pre-processor selectors to do this in the recipe.

Reply:
Thanks for the pointer, Filipe! Done.